docs/en/apiserver.md

```diff
@@ -0,0 +1,215 @@
+# APIServer
+
+To deploy `APIServer` click this: [Deploy OpenMLDB](deploy.md).
+
+Comparison table about JSON datatype and OpenMLDB datatype, and instructions of every function,[RFC](https://github.com/4paradigm/rfcs/blob/main/fedb/api-server.md).
```

Comment:
please help change this sentence to :
Please refer to this [RFC](https://github.com/4paradigm/rfcs/blob/main/fedb/api-server.md) for the comparison of JSON and OpenMLDB data types.